packages/compose/src/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { flowRight } from 'lodash';
+
+export { default as ifCondition } from './if-condition';
```

Comment:
What about `createHigherOrderComponent`? In my opinion, it fits here as well. All HOCs are created using it. It isn't very useful on its own inside `element` package.